tests/Exception_Catch.lap

```diff
@@ -0,0 +1,21 @@
+procedure Test;
+begin
+  raise 'Exception 1';
+end;
+
+begin
+  try
+    try
+      Test();
+    except
+      WriteLn GetExceptionMessage();
+    end;
+  
+    raise 'Exception 2';
+  except
+    WriteLn GetExceptionMessage();
+  end;
+  
+  if GetExceptionMessage() = '' then
```

Comment:
Because the message is stored in `TryStack`, `GetExceptionMessage` outside of a `except`/`finally`-block will never return anything, so maybe we should check for that during compilation and give an error/warning in advance?